resources/js/Pages/Results.vue

```diff
@@ -267,4 +310,25 @@ h2 {
     margin-top: $dimension-layout-xsmall;
 }
 
+p.list-intro {
+    margin-bottom: 0
+}
+
+#description-section {
+    position:relative;
+
+    .description-header {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+
+        h2 {
+            width: 100%;
+        }
+
+        .instructions-button {
+            padding: 6px 13px;
+        }
```

Comment:
I think we might not need these, if we employ `justify-content: space-between`

Reply:
Yes, that's right, and the button padding isn't needed, either - we just use the component's dimensions.

Gone.